great_expectations/core/expectation_configuration.py

```diff
@@ -1268,6 +1268,12 @@ def __eq__(self, other):
             )
         )
 
+    def __lt__(self, other):
+        # Basic comparator for ExpectationType, so that ExpectationSuites can be compared
+        # in a deterministic way. Less-than is determined from Expectation-type
+        # TODO: ensure comparator takes into account kwargs and meta
+        return self.expectation_type < other.expectation_type
```

Comment:
@Shinnnyshinshin This worries me a little -- what is being compared if both are strings?  I am not sure what the meaning of comparison would be as measured in the units of string (lexicographical?) comparison of their `expectation_type` values.